project/AppDependencies.scala

```diff
@@ -6,11 +6,12 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "3.0.0",
-    "uk.gov.hmrc"             %% "simple-reactivemongo"       % "7.30.0-play-27",
-    "com.github.pureconfig"   %% "pureconfig"                 % "0.13.0",
-    "org.typelevel"           %% "cats-core"                  % "2.0.0",
-    "com.beachape"            %% "enumeratum-play"            % "1.5.13"
+    "uk.gov.hmrc"                %% "bootstrap-backend-play-27"  % "3.0.0",
+    "uk.gov.hmrc"                %% "simple-reactivemongo"       % "7.30.0-play-27",
+    "com.github.pureconfig"      %% "pureconfig"                 % "0.13.0",
+    "org.typelevel"              %% "cats-core"                  % "2.0.0",
+    "com.beachape"               %% "enumeratum-play"            % "1.5.13",
+    "com.softwaremill.quicklens" %% "quicklens"                  % "1.6.1"
```

Comment:
Is quicklens only used for tests? if so move to tests

Reply:
thought about it.....then got lazy to reload sbt in IDEA :-) ...will fix next PR